nautobot/core/templates/utilities/templatetags/table_config_form.html

```diff
@@ -10,7 +11,10 @@ <h1>Table Configuration</h1>
             <div class="row align-content-start flex-fill">
                 <div class="col-12">
                     <div class="alert alert-warning">
-                        Adding columns may negatively affect loading times.
+                        Column choices may negatively affect loading times.
+                        <a href="{% static 'docs/user-guide/platform-functionality/configurablecolumns.html' %}" class="btn btn-link btn-sm" target="_blank" rel="noopener">
+                            <i class="mdi mdi-information"></i>
```

Comment:
nit: can we replace `i` with `span`? I'm on a personal crusade of replacing these. functionally it doesn't matter but semantically `i` stands for `italic` text rather than an `icon`

```suggestion
                            <span class="mdi mdi-information"></span>
```